### ISSUE TITLE
add Link component for creating markdown formatted links

### ISSUE THREAD

**Issue by davemeyer** [asynchronous-dev/react-chat-renderer#37]
currently you would have to do something like 
```js
<MarkdownText>
  {`<${'thing.url}|${thing.name}>`}
</MarkdownText>
```
to create a slack link with alt text

it would be simpler to have a
```js
<MarkdownLink url={thing.url} text={thing.name} />
```

**Comment by davemeyer** [comment]
ah, it exists as [Link](https://github.com/asynchronous-dev/react-chat-renderer/blob/master/src/components/Link.ts) already 🤦
